### ISSUE TITLE
Game breaks when going to The Wood

### ISSUE THREAD

**Issue by CommanderTso** [weatherfactory/cultistsimulator-visible#173]
[Issue_173_always_occurs_save.json.zip](https://github.com/weatherfactory/cultistsimulator-visible/files/9075576/save.json.zip)

I went to the wood for the first time, and on getting there found that I couldn't click on the already-revealed Occult Scrap.  Reloading, I tried again and found that I could select one of the hidden cards - it would flip and the others would disappear, but I still couldn't select it.  For some reason, there's also a Funds card in the upper right.

When I save and exit, then reload, it brings be on reload to a point before being in the Mansus.  Not sure if that's typical behavior or not.  I have the various save game backups, happy to forward it all if helpful.  Thanks!

This is on the current MacOS Steam build on Mojave 10.14.6.

**Comment by Chelnoque** [comment]
To clarify, does this problem happen for you every time on this save? If you could check, is it still present if you go with another save?

(@alexiskennedy Funds there are actually in Enroute. Putting Numa rendering layer above Enroute will fix this, but not sure if this is always a wanted behaviour)

**Comment by CommanderTso** [comment]
Hi,

Yes, the issue occurs whenever I load the attached save. I've renamed the save and reuploaded to make its title more clear.

I reloaded a backup save from 4 hours ago, and it has the same problem - I obtained the Wood card, entered the wood, and saw the same as above (with the exception that the Funds card was not present.  That save is attached in this comment.

[issue_173_save_4_hours_earlier.json.zip](https://github.com/weatherfactory/cultistsimulator-visible/files/9076316/issue_173_save_4_hours_earlier.json.zip)

Since my game is somewhat borked at this point, I'll start a new one and see if I can reproduce the issue earlier on, will update here if I can.  Please let me know if there's more a I can do.

**Comment by Chelnoque** [comment]
Wasn't able to reproduce yet. After you encounter this bug in a session, can you please attach Player.log file from the same folder? Maybe there's an exception somewhere, or something.

**Comment by CommanderTso** [comment]
Sure. I tried it again just now.  Attached are two logs - the first was from me trying it and then killing the app after.  Then I realized that it might be helpful for me to attempt to save and exit, since that didn't seem to be working correctly either - that one is also attached.

[play_then_kill_app.log](https://github.com/weatherfactory/cultistsimulator-visible/files/9077242/play_then_kill_app.log)
[play_then_save.log](https://github.com/weatherfactory/cultistsimulator-visible/files/9077243/play_then_save.log)

As a note - the logs on MacOS actually show up in `~/Library/Logs/Weather Factory/Cultist Simulator/`.

**Comment by CommanderTso** [comment]
I made it to The Wood in a new game, and realized that it works fine if I click-and-drag & then let go of the card selected. Prior to this I was just clicking on the card without any drag.

Apologies if I've wasted your time on this - please feel free to close if this is working as desired.

**Comment by alexiskennedy** [comment]
Scott, about once a month I used to get a bug report from someone who didn't realise they had to drag the card to the gate, and thought they were stuck.

Much later I updated the game so that you just needed to drag a card anywhere, not drag it to the gate specifically.

Then I got a bug report - I think from Chel, actually - quite reasonably reporting the new behaviour as broken.

Intended behaviour often isn't obvious!
